docs/databases.rst

```diff
@@ -0,0 +1,149 @@
+Configuring SQL Databases in CommCare
+=====================================
+
+CommCare makes use of a number of logically different SQL databases. These databases
+can be all be a single physical database or configured as individual databases.
+
+By default CommCare will use the `default` Django database for all SQL data.
+
+.. image:: images/django_db_monolith.png
+
+Synlog Data
+~~~~~~~~~~~
+Synclog data may be stored in a separate database specified by the
+`SYNCLOGS_SQL_DB_ALIAS` setting. The value of this setting must be a DB
+alias in the Django `DATABASES` setting.
+
+UCR Data
+~~~~~~~~
+Data created by the UCR framework can be stored in multiple separate databases.
+Each UCR defines an `engine_id` parameter which tells it which configured
+database engine to use. These *engines* are defined in the `REPORTING_DATABASES`
+Django setting which maps the `engine_id` to a Django database alias defined
+in the `DATABASES` setting.
+
+.. code-block:: python
+
+    REPORTING_DATABASES = {
+        'default': 'default',
+        'ucr': 'ucr'
+    }
+
+
+Sharded Form and Case data
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is recommended to have a separate set of databases to store data for Forms
+and Cases (as well as a few other models).
```

Comment:
Is there anything other than form, cases and sms? (currently considering blobs == form since I think a majority of those would be form attachments)

Reply:
ledgers and some SMS stuff